src/common/components/Select.tsx

```diff
@@ -0,0 +1,96 @@
+import {
```

Comment:
what's the difference between using this component or using ui/select directly? if ResizeObserver is needed for all select fields we can maybe add it to ui/select and eliminate this component

Reply:
ui/select is the file created by using the add snippet command. This file has all the composable components that make up a select control. Sure you could piece together the components as you like for each instance you use a select, but that would be tedious. The select component here (src/common/components/Select.tsx) is how I want to use the composable select components and unify all the props into one parent select component. It also has the extra logic needed to make tabs work with the select control.